src/hdf5/atms_netcdf_hdf5_2ioda.py

```diff
@@ -393,6 +400,83 @@ def set_obspace_attributes(VarAttrs):
     return VarAttrs
 
 
+def remapBG(input_files):
+
+    print('Remap ATMS: Backus–Gilbert Inversion method')
+
+    obs_data = init_obs_loc()
+
+    src_ch = [1, 2]
+    tgt_ch = [1, 2]
+    src_bm = 5.2
+    tgt_bm = 3.3
+    nfov = 96
```

Comment:
maybe optional argument saying sensor is ATMS, as nfov 96 is specific to this instrument... also if the src_bm and tgt_bm is in the coefficient file it could be checked here as well.

Reply:
> maybe optional argument saying sensor is ATMS, as nfov 96 is specific to this instrument... also if the src_bm and tgt_bm is in the coefficient file it could be checked here as well.

Hi @BenjaminRuston , do you mean add, such as, "optional.add_argument(
        '-s', '--sensor',"....) for ATMS?